lib/engine/game/base.rb

```diff
@@ -984,6 +984,10 @@ def can_buy_presidents_share_directly_from_market?
         false
       end
 
+      def can_swap_for_presidents_share_directly_from_corporation?
+        true
```

Comment:
1856 hard codes who gets the president's share when the CGR is formed, rather than use share_pool to do it. It worked because of a bug in share_pool. Now that it is fixed, 1856 needs a way to tell share_pool not to hand out the president's share when someone gets enough shares to obtain it.